frontend/packages/console-shared/src/components/drawer/__tests__/Drawer.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Drawer from '../Drawer';
+import { DraggableCore } from 'react-draggable';
+import { CSSTransition } from 'react-transition-group';
+
+describe('DrawerComponent', () => {
+  it('should exist', () => {
+    const wrapper = shallow(<Drawer />);
+    expect(wrapper.find('div').exists()).toBe(true);
+  });
+
+  it('should have default values', () => {
+    const wrapper = shallow(
+      <Drawer>
+        <p>This is content</p>
+      </Drawer>,
+    );
+    expect(wrapper.find('div').exists()).toBe(true);
```

Comment:
In 99% of cases you're not going to want to test markup types. You couple the test and don't really provide anything valuable for "accidental breaks".